sklearn/compose/_column_transformer.py

```diff
@@ -267,15 +279,16 @@ def get_feature_names(self):
             Names of the features produced by transform.
         """
         check_is_fitted(self, 'transformers_')
-        if self._passthrough is not None:
+        passthrough = self._remainder[2]
+        if passthrough is not None:
             raise NotImplementedError(
                 "get_feature_names is not yet supported when having columns"
                 "that are passed through (you specify remainder='drop' to not "
                 "pass through the unspecified columns).")
 
         feature_names = []
         for name, trans, _, _ in self._iter(fitted=True):
-            if trans == 'drop':
+            if trans == 'drop' or name == 'remainder_transformer':
```

Comment:
why should remainder be excluded from feature names?

Reply:
Good point. I updated the PR to remove the restriction.